### ISSUE TITLE
Problems encountered while setting project description.

### ISSUE THREAD

**Issue by timdonovanuk** [adobe/aem-eclipse-developer-tools#69]
Using the AEM project wizard, on the "select or create a server screen", I get the error "Problems encountered while setting project description."

This is using the latest Eclipse, latest plugin, and latest archtype. If I had more info to provide I would..

**Comment by rombert** [comment]
@timdonovanuk  - do you have any related entries in the Eclipse error log?

**Comment by timdonovanuk** [comment]
Yep - apologies I was looking in the Problems tab not the Error log tab:


```
!ENTRY org.eclipse.m2e.logback.appender 2 0 2016-04-26 14:20:50.244
!MESSAGE Manifest foo:bar.core:bundle:0.0.1-SNAPSHOT : Unused Private-Package instructions, no such package(s) on the class path: [!*]

!ENTRY org.eclipse.m2e.logback.appender 2 0 2016-04-26 14:20:50.248
!MESSAGE Manifest foo:bar.core:bundle:0.0.1-SNAPSHOT : The JAR is empty: The instructions for the JAR named foo.bar.core did not cause any content to be included, this is likely wrong

!ENTRY org.eclipse.core.resources 2 568 2016-04-26 14:20:50.590
!MESSAGE Problems encountered while setting project description.
!SUBENTRY 1 org.eclipse.core.resources 2 35 2016-04-26 14:20:50.590
!MESSAGE Nature does not exist: org.eclipse.wst.jsdt.core.jsNature.

!ENTRY org.eclipse.m2e.logback.appender 2 0 2016-04-26 14:20:51.745
!MESSAGE Manifest foo:bar.it.tests:bundle:0.0.1-SNAPSHOT : Unused Private-Package instructions, no such package(s) on the class path: [!*]

!ENTRY org.eclipse.m2e.logback.appender 2 0 2016-04-26 14:20:51.747
!MESSAGE Manifest foo:bar.it.tests:bundle:0.0.1-SNAPSHOT : The JAR is empty: The instructions for the JAR named foo.bar.it.tests did not cause any content to be included, this is likely wrong
```

**Comment by rombert** [comment]
Ah, I see. So it seems that you do not have the Javascript Development Tools installed. Is that correct?

**Comment by timdonovanuk** [comment]
Indeed, I was trying it on Eclipse Mars Java dev edition - I tried installing Web Tools Platform (includes JDT) but that just caused further issues (from http://download.eclipse.org/webtools/repository/mars/).

I've previously tried this plugin with Luna (EE edition) but I get "The server does not support version 2.4 of the J2EE Web module specification." which is why I was testing it on another version of Eclipse.

**Comment by rombert** [comment]
Ah, the 2.4 version thing is something we solved but not yet released ( #67 ).

IIRC you should be able to workaround by setting the JRE version to 1.7 in your content project by setting the `source`/`target` properties for the `maven-compiler-plugin` to 1.7.

**Comment by gabrielwalt** [comment]
Closing this as there hasn't been activity for over a year.
